src/Illuminate/Mail/Mailer.php

```diff
@@ -301,7 +310,7 @@ protected function buildQueueCallable($callback)
             return $callback;
         }
 
-        return (new Serializer)->serialize($callback);
+        return ($this->serializer ?: new Serializer)->serialize($callback);
```

Comment:
Note that I'm not putting this exact code in the constructor so that this is lazy resolved only when we need it so that if someone doesn;t have the super closure library installed, they're all good.